zeppelin-web/src/components/baseUrl/baseUrl.service.js

```diff
@@ -35,6 +35,11 @@ angular.module('zeppelinWebApp').service('baseUrlSrv', function() {
     return wsProtocol + '//' + location.hostname + ':' + this.getPort() + skipTrailingSlash(location.pathname) + '/ws';
   };
 
+  this.getJobManagerWebsocketUrl = function() {
+    var wsProtocol = location.protocol === 'https:' ? 'wss:' : 'ws:';
+    return wsProtocol + '//' + location.hostname + ':' + this.getPort() + skipTrailingSlash(location.pathname) + '/ws/job';
```

Comment:
@bzz 
This code is not currently called.
I have deleted everything associated with it.
However, it seems that you have not deleted the relevant part of the URL.
thank you. I deleted it.